companies/Raytheon-Technologies.md

```diff
@@ -0,0 +1,24 @@
+---
+fortune_rank: 39
+name: Raytheon Technologies Corporation
+tags: ["fortune500", "tech", "defense", "aerospace"]
+blm_statements:
+  - url: https://www.rtx.com/News/2020/06/10/we-must-stand-together
+    date_posted: 2020-06-10
+    type: Company memo
+    raw_text: 
```

Comment:
You may want to use this syntax for a multi-line string.  here's some [documentation](https://www.baeldung.com/yaml-multi-line#folded-style) that might help!

`companies/access-fund.md` is a file that is a good example of how to do this too

```suggestion
    raw_text: >
```